tests/grain_size_plunge_varied_step/statistics

```diff
@@ -0,0 +1,25 @@
+# 1: Time step number
+# 2: Time (years)
+# 3: Time step size (years)
+# 4: Number of mesh cells
+# 5: Number of Stokes degrees of freedom
+# 6: Number of temperature degrees of freedom
+# 7: Number of degrees of freedom for all compositions
+# 8: Number of nonlinear iterations
+# 9: Iterations for temperature solver
+# 10: Iterations for composition solver 1
+# 11: Iterations for Stokes solver
+# 12: Velocity iterations in Stokes preconditioner
+# 13: Schur complement iterations in Stokes preconditioner
+# 14: Average iterations for ODE solver
+ 0 0.000000000000e+00 0.000000000000e+00 1 22 9 9 32 0 0 32 64 63 nan 
+ 1 1.000000000000e+01 1.000000000000e+01 1 22 9 9  2 0 2  2  4  4 nan 
+ 2 2.010000000000e+01 1.010000000000e+01 1 22 9 9  2 0 2  2  4  4 nan 
+ 3 3.030100000000e+01 1.020100000000e+01 1 22 9 9  2 0 2  2  4  4 nan 
+ 4 4.060401000000e+01 1.030301000000e+01 1 22 9 9  2 0 2  2  4  4 nan 
+ 5 5.101005010000e+01 1.040604010000e+01 1 22 9 9  2 0 2  2  4  4 nan 
+ 6 6.152015060100e+01 1.051010050100e+01 1 22 9 9  2 0 2  2  4  4 nan 
+ 7 7.213535210701e+01 1.061520150601e+01 1 22 9 9  2 0 2  2  4  4 nan 
+ 8 8.285670562808e+01 1.072135352107e+01 1 22 9 9  2 0 2  2  4  4 nan 
+ 9 9.368527268436e+01 1.082856705628e+01 1 22 9 9  2 0 2  2  4  4 nan 
+10 1.000000000000e+02 6.314727315639e+00 1 22 9 9  2 0 2  2  4  4 nan 
```

Comment:
So at the moment the output is all nans because of the deal.ii version? Maybe add a comment about that somewhere? And did it actually change the number of iterations?